test/topology_experimental_raft/test_tablets_intranode.py

```diff
@@ -120,3 +140,54 @@ async def test_intranode_migration(manager: ManagerClient):
     assert len(rows) == key_count
     for r in rows:
         assert r.c == r.pk
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_crash_during_intranode_migration(manager: ManagerClient):
+    cmdline = [
+        '--logger-log-level', 'tablets=trace',
+        '--logger-log-level', 'database=trace',
+        '--commitlog-sync', 'batch', # So that ACKed writes are not lost on crash
+    ]
+    servers = [await manager.server_add(cmdline=cmdline)]
+
+    await manager.api.disable_tablet_balancing(servers[0].ip_addr)
+
+    cql = manager.get_cql()
+
+    await cql.run_async("CREATE KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1}"
+                        " AND tablets = {'initial': 4};")
+    await cql.run_async("CREATE TABLE test.test (pk int PRIMARY KEY, c int);")
+
+    finish_writes = await start_writes(cql, "test", "test", ignore_errors=True)
+
+    tablet_token = 0 # Choose one tablet, any of them
+    replica = await get_tablet_replica(manager, servers[0], 'test', 'test', tablet_token)
+
+    src_shard = replica[1]
+    dst_shard = src_shard ^ 1
+
+    await manager.api.enable_injection(servers[0].ip_addr, 'crash-in-tablet-write-both-read-new', one_shot=True)
+
+    migration_task = asyncio.create_task(manager.api.move_tablet(servers[0].ip_addr, "test", "test",
+                                                replica[0], src_shard, replica[0], dst_shard, tablet_token))
+
+    s0_logs = await manager.server_open_log(servers[0].server_id)
+    await s0_logs.wait_for('crash-in-tablet-write-both-read-new hit')
+    await manager.rolling_restart(servers)
```

Comment:
this could introduce a racing condition. where `manager.rolling_restart()` calls into `self.server_stop_gracefully(s.server_id)` which in turn calls into `ScyllaServer.stop_gracefully()`. and the server is the one which crashes because the failure injection. 

but `ScyllaServer.stop_gracefully()` sends `SIGTERM` to the server, but at that moment the server process already exits with non-zero status code.

so we have failures like https://jenkins.scylladb.com/job/scylla-master/job/next/7613/testReport/junit/(root)/test_tablets_intranode/Build___aarch64___Unit_Tests_aarch64___test_crash_during_intranode_migration/